src/com/ableton/Pyenv.groovy

```diff
@@ -46,8 +46,12 @@ class Pyenv implements Serializable {
     VirtualEnv venv = new VirtualEnv(script, randomSeed)
     int result
     script.withEnv(["PYENV_VERSION=${pythonVersion}"]) {
-      List installCommands = [
-        "export PYENV_ROOT=${pyenvRoot}",
+      List installCommands = []
+      if (!script.env.PYENV_ROOT) {
+        // Only export PYENV_ROOT if it isn't part of the Jenkins environment
+        installCommands.add("export PYENV_ROOT=${pyenvRoot}")
```

Comment:
I'm not sure I follow you here. Why do you say, "if it might get ignored"? If `pyenvRoot` is added as the optional argument to either function in the `pyenv` singleton, it'll be set here, not ignored.

The purpose of this line is to just prevent exporting something which is already in the environment.

Reply:
The `pyenvRoot` field won't be used here if `script.env.PYENV_ROOT` is true-ish. That's what I mean by "ignored".

Reply:
I think that the idea here was that we didn't want to have "conflicting" values, but come to think of it, maybe we _do_ always want to export `PYENV_ROOT` because we have determined the "safe" value. Fixed in 2d0d2c7 (which means we'll drop that commit). I'll do some tests as well.